source/includes/gho/_api.md

```diff
@@ -0,0 +1,305 @@
+updated_at | [DateTime](#datetime) | 
+
+## CustomFieldValue
+A Custom Field Value Record
+
+Field | Type | Description
+--------- | ----------- | -----------
+created_at | [DateTime](#datetime) | 
+custom_field | [CustomField](#customfield) | 
+id | [ID](#id) | 
+updated_at | [DateTime](#datetime) | 
+value | [Value](#value) |     A different type of value will be stored based upon the field type of the [CustomField](#customfield).  Some types    have the data stored as a nested object.  Note that the type is a scalar named [Value](#value).  Even though    it appears to be an object, you are not able to use GraphQL to determine its shape.  
+value_updated_at | [DateTime](#datetime) | The time of the most recent update to this field.
+
+## Department
+    Represents a single department in your company.  Employees can belong only belong to a single Department.  It is    possible for employees to not belong to any Department.  Departments are used in a variety of ways in Greenhouse    Onboarding, including permissions and onboarding plans  
+
+Field | Type | Description
+--------- | ----------- | -----------
+created_at | [DateTime](#datetime) | 
+description | [String](#string) | 
+id | [ID](#id) | 
+name | [String](#string) | 
+updated_at | [DateTime](#datetime) | 
+
+## Document
+Represents a single document attached to an [Employee](#employee).
+
+Field | Type | Description
+--------- | ----------- | -----------
+created_at | [DateTime](#datetime) | 
+file | [File](#file) | Contains the file payload.
+id | [ID](#id) | 
+updated_at | [DateTime](#datetime) | 
+
+## Employee
+A single Employee that works for your company.
+
+Field | Type | Description
+--------- | ----------- | -----------
+about | [String](#string) |     A brief description of the employee.  This information is displayed on both the employee's profile and is also    featured prominently in the Welcome Experience for any new hires that report to them.  
+created_at | [DateTime](#datetime) | 
+custom_field_values | [CustomFieldValue](#customfieldvalue) |     A list of all other profile information for this employee.  Administrators can configure these fields on the    [Settings > Custom Fields](https://onboarding.greenhouse.io/settings/fields) page.  
+date_of_birth | [Date](#date) | Note that only administrators can see the birth year for employees
+date_of_termination | [Date](#date) | This information is only available on terminated employees
+department | [Department](#department) | 
+documents | [Document](#document) |     These are documents that were either came over from Greenhouse Recruiting, were attached directly to the employee    profile, or attached to a task.  This does _not_ include E-Signature requests.  
+email | [String](#string) | The employee's work email.  They need this in order to sign in
+employment_status | [EmploymentStatus](#employmentstatus) | 
+first_name | [String](#string) | 
```

Comment:
would it make sense to group similar fields like `first_name` and `last_name` or do we want to keep it alphabetical?

Reply:
Sorting might be tough.  The schema exports these things sorted alphabetically, and my doc generator just iterates over the fields in the schema.